src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -2191,8 +2191,7 @@ internal static void LogScriptBlockEnd(ScriptBlock scriptBlock, Guid runspaceId)
         internal bool HasCleanBlock { get => AstInternal.Body.CleanBlock != null; }
     }
 
-    [Serializable]
-    internal class ScriptBlockSerializationHelper : ISerializable, IObjectReference
```

Comment:
I don't see the corresponding interface methods from `ISerializable, IObjectReference` got removed.

Reply:
fixed